fractal_tasks_core/cellvoyager/metadata.py

```diff
@@ -174,8 +185,14 @@ def read_mrf_file(mrf_path: str):
     mrf_frame = pd.DataFrame(columns=mrf_columns)
 
     mrf_xml = ElementTree.parse(mrf_path).getroot()
+    # fixme: get bts:ColumnCount="48" bts:RowCount="32"
```

Comment:
> `# fixme: get bts:ColumnCount="48" bts:RowCount="32"`

Is this still useful/relevant?

Reply:
Ah, that's a leftover comment, not needed anymore